crates/milli/src/index.rs

```diff
@@ -177,10 +177,10 @@ pub struct Index {
     /// Maps the document id, the facet field id and the strings.
     pub field_id_docid_facet_strings: Database<FieldDocIdFacetStringCodec, Str>,
 
-    /// Maps an embedder name to its id in the arroy store.
+    /// Maps an embedder name to its id in the hannoy store.
     pub(crate) embedder_category_id: Database<Unspecified, Unspecified>,
-    /// Vector store based on arroy™.
-    pub vector_arroy: arroy::Database<Unspecified>,
+    /// Vector store based on hannoy™.
+    pub vector_hannoy: hannoy::Database<Unspecified>,
```

Comment:
To discuss, but we should only keep the hannoy one and remap the types when we need. We can do that in the `VectorStore` wrapper ex-`ArroyWrapper`.